packages/scss/src/components/list/component.scss

```diff
@@ -36,13 +38,23 @@
 			margin: 0;
 		}
 
+		.list-item-content-helper {
+			margin: 0;
+			color: var(--palettes-grey-600);
+			font-size: var(--sizes-S-fontSize);
+			line-height: var(--sizes-S-lineHeight);
+		}
+
 		.list-item-actions {
-			padding: var(--components-list-padding);
 			transition-duration: var(--commons-animations-durations-fast);
 			transition-property: all;
```

Comment:
I'm not sure what's being transitioned here but `all` might be overkill.

Reply:
It looks like legacy code used for action opacity. -> deleted.